model/annotations/task_annotations.go

```diff
@@ -204,6 +204,48 @@ func UpdateAnnotation(a *TaskAnnotation, userDisplayName string) error {
 	return errors.Wrapf(err, "problem adding task annotation for '%s'", a.TaskId)
 }
 
+func AddToAnnotation(a *TaskAnnotation, userDisplayName string) error {
```

Comment:
Should add function comment. Also preference for PatchAnnotation

Reply:
done